src/lat/sausages.h

```diff
@@ -222,15 +232,20 @@ class MinimumBayesRisk {
   // paper.  We sort in reverse order on the second member (posterior), so more
   // likely word is first.
 
-  std::vector<std::pair<BaseFloat, BaseFloat> > times_;
+  std::vector<std::vector<std::pair<BaseFloat, BaseFloat> > > times_;
+  // The average start and end times for words in each confusion-network bin.
+  // This is like an average over arcs, of the tau_b and tau_e quantities in
```

Comment:
This comment looks like it was copied from below.  Is it correct?
The indexing is not clear to me (also the original code).

Reply:
Yes, the comment is correct. What used to be `times_` is now called `sausage_times_`.

```
times_          -> tau_b(q, w), tau_e(q, w)
sausage_times_  -> tau_b(q), tau_e(q)
```